catalog/internal/server/openapi/api_model_catalog_service_service.go

```
@@ -27,6 +28,10 @@ func (m *ModelCatalogServiceAPIService) GetAllModelArtifacts(ctx context.Context
 		return notFound("Unknown source"), nil
 	}
 
+	if newName, err := url.PathUnescape(name); err == nil {
+		name = newName
+	}
```

If `err` is not nil should we return an error response instead of silently using the original value?

---

I didn't want to because the escaped name might be valid.